src/torrent.cpp

```diff
@@ -10558,22 +10558,23 @@ namespace libtorrent
 
 		// activity time
 #ifndef TORRENT_NO_DEPRECATE
-		st->finished_time = finished_time();
-		st->active_time = active_time();
-		st->seeding_time = seeding_time();
+		// cast to seconds in case that internal values doesn't have ratio<1>
+		st->finished_time = int(duration_cast<seconds>(finished_time()).count());
```

Comment:
no, that would be wrong if we change the rate to non 1.

The interface except real seconds, that why I cast to real "seconds" instead of seconds32. Like the comment already tell.